.github/CODEOWNERS

```
@@ -40,6 +40,7 @@
 # See: https://github.com/apache/airflow/blob/main/airflow-core/src/airflow/ui/public/i18n/README.md#43-engaged-translator
 airflow-core/src/airflow/ui/public/i18n/locales/ar/ @shahar1 @hussein-awala # + @ahmadtfarhan
 airflow-core/src/airflow/ui/public/i18n/locales/de/ @jscheffl # + @TJaniF @m1racoli
+airflow-core/src/airflow/ui/public/i18n/locales/es/ @aoelvp94
```

Unfortunately non committer people cannot be code owner. You can be a sponsored translation owner but this needs to go through the mailing list. (Committer sponsoring you will do that). You can find more information in our i18n policy. https://github.com/apache/airflow/blob/main/airflow-core/src/airflow/ui/public/i18n/README.md

---

maybe could @bbovenzi be the reviewer and owner ? (I know you know spanish :P). Glad to receive feedback

---

...then it would be...
```suggestion
airflow-core/src/airflow/ui/public/i18n/locales/es/ @bbovenzi # + @aoelvp94
```

---

Yep, but waiting for @bbovenzi consent for adding him :)

---

Yeah that works for me for now